src/schemas/airdrops.graphql

```diff
@@ -122,6 +122,10 @@ type Action @entity(immutable: true) {
   """
   claimRecipient: Bytes
   """
+  Claim action data: to.
```

Comment:
Non-explanatory description - we have to update it to clarify how it is different from `claimRecipient`, and that's it's only available in Airdrops v2.0 and later.